### PR TITLE
fix(core): fix swizzle legend typo

### PR DESCRIPTION
**Pull request by DigiPie**
## Motivation

Fix swizzle status description typo.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yep

## Test Plan

```bash
yarn test packages/docusaurus/src/commands/swizzle
```

## Related PRs

- Follows #6243 - @Josh-Cena thanks for implementing this feature, love the new DX!

### CODE REVIEW

File: packages/docusaurus/src/commands/swizzle/tables.ts [hunk 1]
```diff
@@ -74,11 +74,11 @@ function actionsTable(): string {
       logger.code('--wrap'),
       `
 Creates a wrapper around the original theme component.
-Allows rendering other components before/after the original theme component.
+Allows rendering of other components before/after the original theme component.
```
Comment (Josh-Cena) [inline packages/docusaurus/src/commands/swizzle/tables.ts:77]:
```suggestion
Allows rendering other components before/after the original theme component.
```

This is actually correct 😄

**Comment by Josh-Cena** [comment]
Thanks! Most of the credit goes to @slorber — he wrote most of this stuff, I just did the initial work 😄

**Comment by DigiPie** [comment]
> Thanks! Most of the credit goes to @slorber — he wrote most of this stuff, I just did the initial work 😄

I see, thanks @slorber 😃

**Comment by slorber** [comment]
np :) thanks for fixing my typos :D

**Pull request merged**